chat/chat-impl/impl/src/webapp/WEB-INF/components.xml

```diff
@@ -21,8 +21,18 @@
 				  init-method="init"
 				  destroy-method="destroy">
 				<property name="sessionFactory" ref="org.sakaiproject.springframework.orm.hibernate.GlobalSessionFactory"/>
-				<property name="entityManager" ref="org.sakaiproject.entity.api.EntityManager"/>
 				<property name="defaultChannelSettings" ref="defaultChatChannelSettings"/>
+				<property name="userDirectoryService" ref="org.sakaiproject.user.api.UserDirectoryService"/>
+				<property name="serverConfigurationService" ref="org.sakaiproject.component.api.ServerConfigurationService"/>
+				<property name="presenceService" ref="org.sakaiproject.presence.api.PresenceService"/>
+				<property name="sessionManager" ref="org.sakaiproject.tool.api.SessionManager" />
+				<property name="usageSessionService" ref="org.sakaiproject.event.api.UsageSessionService"/>
+				<property name="formattedText" ref="org.sakaiproject.util.api.FormattedText"/>
+				<property name="timeService" ref="org.sakaiproject.time.api.TimeService"/>
```

Comment:
Yeah... I have the same improvement in my blackboard with pending tasks. I'm going to take a look at this...